gitops/argocd/README.md

```diff
@@ -0,0 +1,43 @@
+# Argo CD registration
+
+This directory contains the logic used for registering the clusters to Argo CD.
+
+## Run
+
+The registration is thought to be triggered from [Pipelines as Code](https://pipelinesascode.com/) and a [Tekton PipelineRun](../pac/.tekton/argo-registration.yaml) is provided for the purpose in the .tekton directory.
+
+Alternatively the registration can be performed by manually calling the registration script in the image directory:
+
+```console
+ARGO_URL="https://argoserver.com" ARGO_USER="user" ARGO_PWD="xxxxxxxxx" DATA_DIR="/workspace" ./register.sh
+```
+
+DATA_DIR should point to a directory with a fork of this repository including
```

Comment:
I think that [this slide](https://speakerdeck.com/spesnova/introduction-to-kustomize?slide=73) is representative of a common practice with GitOps. Could you please share references for a different approach so that I can do some reading?